erc20-load-test-tool/script/readme.md

```diff
@@ -30,5 +30,10 @@ $ source .env
 
 3. Run the script
 ```bash
-$ python3 run-dapp.py
```

Comment:
Discussed with @ramkri123 . It would be good to create a .sh having this `nohup` command. This will enforce the correct way to invoke this tool.

Reply:
Done, added nohup in bash script along with checking java version as suggested by @ramkri123